### ISSUE TITLE
bug: Unable to update profile details without filling change password form values

### ISSUE THREAD

**Issue by muhsin-k** [chatwoot/chatwoot#2053]
**How to reproduce?**
1. Go to the profile settings page
2. Change the display name
3. Press "Update profile"
4. It will show an error "Confirm password should match the password"

**Preview**
<img width="1206" alt="Screenshot 2021-04-05 at 8 49 11 PM" src="https://user-images.githubusercontent.com/12408980/113590613-bb762c00-95e7-11eb-8482-e0e3c1a7a284.png">
